.betterer.results

```diff
@@ -5397,6 +5397,9 @@ exports[`no undocumented stories`] = {
     "packages/grafana-ui/src/components/ThemeDemos/ThemeDemo.story.tsx:5381": [
       [0, 0, 0, "No undocumented stories are allowed, please add an .mdx file with some documentation", "5381"]
     ],
+    "packages/grafana-ui/src/components/UnitPicker/UnitPicker.story.tsx:5381": [
+      [0, 0, 0, "No undocumented stories are allowed, please add an .mdx file with some documentation", "5381"]
+    ],
```

Comment:
Do we want an mdx file here? If not, I think we should rethink this betterer rule in general

Reply:
An empty MDX file serves no one. Ash and I have discussed removing the betterer rule, but I didn't want to do that in this PR.

Reply:
This one wasn't empty, it was broken with runtime error messages in storybook. The MDX file only had argtypes which the auto generated docs have so seemed simpler to just delete it for now. 🤷